src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -136,7 +142,7 @@ public MaxwellKafkaProducerWorker(MaxwellContext context, Properties kafkaProper
 			this.topic = "maxwell";
 		}
 
-		this.interpolateTopic = kafkaTopic.contains("%{");
+		this.interpolateTopic = this.topic.contains("%{");
```

Comment:
Could we just instantiate with a null topic and make sure it doesn't explode? I'm assuming the previous code would...